src/services/mail.js

```diff
@@ -0,0 +1,49 @@
+/* eslint-disable consistent-return */
+/* eslint-disable no-console */
+const nodemailer = require('nodemailer');
+require('dotenv').config();
+
+const emailForRegistering = async (firstName, lastName, username, email) => {
```

Comment:
can we change function name here to start with a verb so smth like `sendEmailForRegisteringUsers`